src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Blob.txt

```diff
@@ -0,0 +1,116 @@
+#SETUP: Blob, EnableExpressionChaining
+
+// Blob handler creates variables
+// - blob as BlobValue
+// - str as StringValue
+
+// Cannot run a test like Blob("abc") as it would return a BlobValue
+// and we can't call ToExpression() of such an object (NotImplementedException)
+
+>> Text(Blob("abc"))
```

Comment:
>abc [](http://example.com/codeflow?start=14&length=3)

Should we force the argument to blobs to be valid URIs?

Reply:
Like this?
`Set(id, AdRM("abc")); Text(Blob(id))`